src/modules/analytics/rollbar.ts

```diff
@@ -1,28 +1,22 @@
 import { env } from 'decentraland-commons'
+import { isDevelopment } from 'lib/environment'
 
 let _rollbarConfig = {
   accessToken: env.get('REACT_APP_ROLLBAR_ACCESS_TOKEN', ''),
   captureUncaught: true,
   payload: {
-    environment: env.get('NODE_ENV'),
+    environment: env.get('REACT_APP_ENVIRONMENT'),
```

Comment:
What do you think of using the exported variable `development` from `lib/environment` here.